environment.yml

```diff
@@ -6,14 +6,9 @@ channels:
 
 dependencies:
   - python=3.8
-  - clang
-  - cmake
-  - ninja
   - pip
   - matplotlib
   - openmpi
   - mpi4py
-  - blas
   - mkl
   - numba
-  - icc_rt
```

Comment:
I would keep this, since numba can use it to make trascendent functions a lot faster

Reply:
are you sure that Numba automatically picks it up? In any case this environment is only for testing, no?

Reply:
yes it should be [automatically picked up ](https://numba.pydata.org/numba-doc/latest/user/performance-tips.html#intel-svml) 

it is for testing true, but we might leave it there as a pointer to what we need in the conda environment ?

Reply:
ah good.
I should add it to the conda recipe, btw... I think I never put it in